query_server/query/src/instance.rs

```diff
@@ -250,6 +250,7 @@ pub async fn make_cnosdbms(
         .with_query_tracker(query_tracker)
         .with_func_manager(Arc::new(func_manager))
         .with_stream_provider_manager(stream_provider_manager)
+        .with_auth_enable(options.query.auth_enabled)
```

Comment:
".with_query_execution_factory(query_execution_factory)" already has "auth_enabled", don't pass it again